synapse/tests/tools/test_dmon.py

```diff
@@ -0,0 +1,16 @@
+
+from synapse.tools.dmon import getArgParser
+import pytest
```

Comment:
While travis-ci ( and maybe even AppVeyor ) have pytest present in their test environments, it's also expected that a clean checkout should run correctly on other platforms.  Minimizing dependancies is a huge goal of synapse so I think in this case adding an entire 3rd party dependency for this one test is probably not compelling enough :)  For consistency sake, would you mind implementing a quick unit test that lives in the same module but extends SynTest ( from synapse.tests.common ) using only the "unittest" builtins and/or synapse itself?  :)